content/webapp/pages/search/index.tsx

```
@@ -226,12 +242,51 @@ export const SearchPage: NextPageWithLayout<Props> = ({
     setLink(link);
   }, [query]);
 
+  async function fetchWorks() {
+    try {
+      const worksResults = await getWorks({
+        params,
+        pageSize: 1,
+        toggles: data.toggles,
+      });
+      const works = getQueryResults({
+        categoryName: 'works',
+        queryResults: worksResults,
+      });
+      setClientSideWorks(works);
+    } catch (e) {
+      return undefined;
```

Should we throw an Error/something to log?